tests/requirements.txt

```diff
@@ -1,21 +1,28 @@
-attrs==21.2.0
+attrs==23.1.0
 black==21.9b0
 certifi==2021.10.8
 chardet==4.0.0
 charset-normalizer==2.0.7
 click==8.0.3
 colorama==0.4.4
 decorator==5.1.0
-frictionless==4.20.2
+frictionless==5.13.1
+humanize==4.6.0
 idna==3.3
 isodate==0.6.0
+Jinja2==3.1.2
```

Comment:
Are all these extra requirements dependencies of Frictionless v5?

Reply:
I have created venv using `requirements.txt` from the main branch and simply upgraded `frictionless` package to the newest version and this is the list I have got.